src/components/NcAppNavigation/NcAppNavigation.vue

```diff
@@ -66,7 +66,7 @@ emit('toggle-navigation', {
 			@keydown.esc="handleEsc">
 			<slot />
 			<!-- List for Navigation li-items -->
-			<ul class="app-navigation__list">
+			<ul v-if="hasListSlotItems" class="app-navigation__list">
```

Comment:
Is `ul.app-navigation__list` different from `NcAppNavigationList`?

Reply:
Also stumbled over this one, could we not just use `NcAppNavigationList` here instead?

Reply:
> Is `ul.app-navigation__list` different from `NcAppNavigationList`?

Yes slightly. The difference is `ul.app_navigation__list` always fills the height that is not already taken up by its siblings inside `nav.app-navigation__content` but when we have separate lists inside the nav we want don't want that e.g.

![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/24800714/1bf48697-c94a-4715-8262-375f455cf13b)

so we adjust the styles in NcAppNavigationList to keep the styling we want:

![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/24800714/6343f9ea-af20-47c5-b8ee-a8c277b9707e)

The `.app_navigation__list` styles also target `.app_navigation__content > ul` which we should keep to not break anything, so we also increase the specificity to have the styles apply correctly